src/Rest/Payloads/UpdatePlayerPayload.cs

```diff
@@ -11,7 +11,7 @@ namespace Victoria.Rest.Payloads;
     [property: JsonPropertyName("identifier")]
     string Identifier = default,
     [property: JsonPropertyName("position")]
-    int Position = default,
+    double Position = default,
```

Comment:
keep it int

Reply:
Cannot be int as TotalMiliseconds is Double